android/config.json

```diff
@@ -1,12 +1,8 @@
 {
-    "copyFiles": [
-        "UtilsPlugin.java"
-    ],
-    "libraries": [
-    ],
-    "jars": [
-        "google-basement.jar"
-    ],
-    "injectionXML": "manifest.xml",
-	"injectionXSL": "manifest.xsl"
+	"copyFiles": [
```

Comment:
this file still has tabs. everything else looks good

Reply:
Should config files also have 2 spaces?

Reply:
yeah. But Let's ignore for now in the interest of closing these